specification/storage/resource-manager/Microsoft.Storage/stable/2023-01-01/storage.json

```diff
@@ -5045,6 +5229,69 @@
       },
       "description": "Filters limit replication to a subset of blobs within the storage account. A logical OR is performed on values in the filter. If multiple filters are defined, a logical AND is performed on all filters."
     },
+    "ObjectReplicationRestorePoints": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ObjectReplicationRestorePoint"
+          },
+          "description": "The replication restore points of storage account."
+        }
+      },
+      "description": "List of storage account object replication restore points."
+    },
+    "ObjectReplicationRestorePointMarker": {
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "Restore point id is auto-generated for each new restore point marker of account."
+        },
+        "label": {
+          "type": "string",
+          "description": "Label of restore point marker, for reference."
+        },
+        "timestamp": {
+          "type": "string",
+          "readOnly": true,
+          "format": "date-time",
+          "description": "Timestamp of the restore point creation."
+        }
+      },
+      "description": "The replication restore point marker of storage account."
+    },
+    "ObjectReplicationRestorePoint": {
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "Restore point id is auto-generated for each new restore point of account."
+        },
+        "label": {
+          "type": "string",
+          "description": "Label of restore point, for reference."
+        },
+        "sourceAccount": {
```

Comment:
I don't see any place to put the source account. (Not sure if missed it?)
So how server get the value and return it?

Reply:
reverted this change from PR it is not in scope for this release.